ddtrace/profile/__init__.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+import pkg_resources
+
+from ddtrace.profile import _build
+
+try:
+    __version__ = pkg_resources.get_distribution(__name__).version
+except pkg_resources.DistributionNotFound:
+    # package is not installed
+    __version__ = "unknown"
+
+
+def _not_compatible_abi():
+    raise ImportError("Python ABI is not compatible, you need to recompile this module")
+
+
+if _build.compiled_with["major"] == 3 and _build.compiled_with["minor"] == 7:
```

Comment:
this might be easier to understand if we use a tuple to compare.

```python
compiled_with (_build.compiled_with["major"], _build.compiled_with["minor"], _build.compiled_with["micro"])
if compiled_with >= (3, 7, 3) and sys.version_info < (3, 7, 3):
    _not_compatible_abi()
```